starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ItemObject.java

```diff
@@ -0,0 +1,20 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by student on 10/25/16.
+ */
+public class ItemObject {
+    private String mItem;
+
+    public ItemObject(String item) {
+        mItem = item;
+    }
+
+    public String getItem() {
+        return mItem;
+    }
+
+    public void setItem(String item) {
+        mItem = item;
```

Comment:
Totally fine to have a setter here, but I just want to make sure you know you don't always need to have setters. Sometimes you don't want an object's values to change after it's created, so you might prefer not to provide any setters.